ansible/roles/monit/handlers/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+  - name: monit
+    shell: monit stop all && monit reload && monit start all
```

Comment:
i don't know. i kind of like this since it is less ambiguous to me. i don't need to know some fine detail of monit to know what it does.